src/log-manager/src/nginx/list_logs.lua

```diff
@@ -43,19 +42,19 @@ end
 
 local log_query_param = "?username="..username.."&framework-name="..framework_name..
   "&pod-uid="..pod_uid.."&taskrole="..taskrole.."&token="..token
-local path = "/usr/local/pai/logs/"..username.."/".. framework_name.."/".. taskrole.."/"..pod_uid.."/"
+local log_dir = "/usr/local/pai/logs/"..username.."/".. framework_name.."/".. taskrole.."/"..pod_uid.."/"
 local path_prefix = "/api/v1/logs/"
 
 local ret = {}
 
-if not is_dir(path) then
+if not util.is_path_under_log_folder(log_dir) or not path.isdir(log_dir) then
```

Comment:
could you unify the name `folder` and `dir`, and differ from `path`?

Reply:
Fixed